clinica/pydra/machine_learning_spatial_svm/spatial_svm_cli.py

```diff
@@ -0,0 +1,98 @@
+from typing import Optional
+
+import click
+
+import clinica.pydra.engine_utils as pydra_utils
+import clinica.pydra.machine_learning_spatial_svm.pipeline as pydra_machine_learning_spatial_svm
+from clinica.pipelines import cli_param
+from clinica.pipelines.engine import clinica_pipeline
+
+pipeline_name = "pydra-machinelearning-prepare-spatial-svm"
```

Comment:
Small thing, but given the structure of this name, I would also expect a "-" between "machine" and "learning".